bin/strata-cli/src/constants.rs

```diff
@@ -26,20 +26,20 @@ pub const PW_SALT_LEN: usize = 16;
 /// Length of nonce in bytes
 pub const AES_NONCE_LEN: usize = 12;
 /// Length of seed in bytes
-pub const SEED_LEN: usize = 32;
+pub const SEED_LEN: usize = 16;
 /// AES-256-GCM-SIV tag len
 pub const AES_TAG_LEN: usize = 16;
 
-pub const NETWORK: Network = Network::Signet;
+pub const DEFAULT_NETWORK: Network = Network::Signet;
 pub const BRIDGE_STRATA_ADDRESS: &str = "0x000000000000000000000000000000000B121d9E";
```

Comment:
bridge address is now
5400000000000000000000000000000000000001

Reply:
Should this be part of the params?